modules/virtual-cluster/variables.tf

```
@@ -145,3 +145,27 @@ variable "cloudwatch_log_group_kms_key_id" {
   type        = string
   default     = null
 }
+
+variable "cloudwatch_log_group_name" {
+  description = "The custom name for the AWS CloudWatch log group. If 'cloudwatch_log_group_use_name_prefix' is false or not provided, this custom name will be used. If empty, a default name will be generated."
```

From the provider docs:
```suggestion
  description = "The name of the log group. If a name is not provided, the default name format used is: `/emr-on-eks-logs/emr-workload/<NAMESPACE>`"
```